cypress/integration/openinigNewFile.spec.ts

```diff
@@ -0,0 +1,21 @@
+
+describe('opening new tab', function () {
+    beforeEach(() => {
+        cy.visit('https://the-internet.herokuapp.com/windows')
+    })
+
+    it('opening new tab by removing attribute target pracitice', () => {
+        const newWindow = 'New Window'
+        cy.contains('Click Here').invoke('removeAttr', 'target').click()
+        cy.url().should('have.text', '/windows/new')
+        cy.get(newWindow).should('have.text', 'New Window')
```

Comment:
when we change the code we need to run it to make sure it's fine
cy.get('New Window') will not work!